src/google/protobuf/stubs/atomicops.h

```diff
@@ -194,14 +194,6 @@ Atomic64 Release_Load(volatile const Atomic64* ptr);
 #elif defined(GOOGLE_PROTOBUF_OS_AIX)
 #include <google/protobuf/stubs/atomicops_internals_power.h>
 
-// Apple.
```

Comment:
Could you try this: https://github.com/google/protobuf/blob/master/protoc-artifacts/build-protoc.sh#L204?